src/renderer/families/tron/Votes/index.js

```diff
@@ -61,6 +62,24 @@ const useNewVotingDate = () => {
   return nextVotingDate;
 };
 
+// @TODO move this to common
```

Comment:
🙏

Reply:
I'll move all the necessary hooks once all the flows are done if that is ok with you